js/src/personal/maliha/Maliha.page.tsx

```diff
@@ -1,12 +1,52 @@
 import { Text, MantineProvider } from '@mantine/core'
-import { Menu } from './ModalMenu.tsx'
 import { theme } from './theme.ts'
+import { Menu } from './ModalMenu.tsx'
+import { Social } from './Social.tsx'
 import styles from './Maliha.module.css'
 
 export function MalihaPage() {
   return (
     <MantineProvider theme={theme}>
       <div className={styles.body}>
+        <link
```

Comment:
Can you put the font linking in another file/component? It make this one harder to read.